pkg/controller/release/migration.go

```diff
@@ -116,10 +116,19 @@ func (c *Controller) migrateCapacityTargets(relName, namespace string, selector
 		ct.Labels[shipper.MigrationLabel] = "true"
 		_, err = clusterClientsets.GetShipperClient().ShipperV1alpha1().CapacityTargets(ct.Namespace).Update(ct)
 		if err != nil {
-			return err
+			ctErrors.Append(err)
+		}
+	}
+	if ctErrors.Flatten() == nil && shouldTag {
+		// update initial object with migrated label
+		initialCt.Labels[shipper.MigrationLabel] = "true"
+		_, err = c.clientset.ShipperV1alpha1().CapacityTargets(namespace).Update(initialCt)
+		if err != nil {
+			klog.Error(err)
```

Comment:
nit: should we something before logging the error to make it clearer? E.g. `klog.Errorf("error updating capacity targets %v": err)`

If you decided to change it, I would change it also for `TrafficTargets` and `InstallationTargets`